contrib/mathtex-script-type/README.md

```diff
@@ -0,0 +1,16 @@
+# `math/tex` Custom Script Type Extension
+
+This is an extension to automatically display code inside `script` tags with `type=math/tex` using KaTeX.
+This script type is commonly used by MathJax, so this can be used to support compatibility with MathJax.
+
+### Usage
+
+This extension isn't part of KaTeX proper, so the script should be separately
+included in the page.
+This include should be loaded after all `script` blocks you want to render.
+
+```html
+<script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/contrib/mathtex-script-type.min.js" integrity="sha384-o+v+EkJWQmZj7XwHBxehTGJKE18182WyyN2glZMTPw9g5XxjN1uwrquNuMX/NJiF"></script>
+```
```

Comment:
Could you expand on this script a bit more?  I assume that I need to load KaTeX somewhere too.

Maybe something along these lines:
```html
<html>
   <head>
       <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/katex.min.css" integrity="sha384-wITovz90syo1dJWVh32uuETPVEtGigN07tkttEqPv+uR2SE/mbQcG7ATL28aI9H0" crossorigin="anonymous">
       <script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/katex.min.js" integrity="sha384-/y1Nn9+QQAipbNQWU65krzJralCnuOasHncUFXGkdwntGeSvQicrYkiUBwsgUqc1" crossorigin="anonymous"></script>
   </head>
   <body>
      <script type="math/tex">x+\sqrt{1-x^2}</script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/contrib/mathtex-script-type.min.js" integrity="sha384-o+v+EkJWQmZj7XwHBxehTGJKE18182WyyN2glZMTPw9g5XxjN1uwrquNuMX/NJiF"></script>
   </body>
</html>
```